gtfs-realtime/spec/en/reference.md

```diff
@@ -439,7 +439,7 @@ TripDescriptor.route_id cannot be used within an Alert EntitySelector to specify
 
 #### Fields
 
-| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
```

Comment:
Can you squash all your commits? It is pretty hard to follow what is actually changed? And if the table is just moved to another place and reformatted put it in a separate commit, please.